### PR TITLE
Path error

### LINKED ISSUE #21019: Fix path
[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 164d7efe-01bf-0b26-7a23-f10704f338f5
* Version Independent ID: b3e2bc87-1582-2d34-c7c2-7c8e901ac771
* Content: [Introduction to Razor Pages in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/razor-pages/?view=aspnetcore-5.0&tabs=visual-studio)
* Content Source: [aspnetcore/razor-pages/index.md](https://github.com/dotnet/AspNetCore.Docs/blob/master/aspnetcore/razor-pages/index.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-razorpages**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by wuyuansushen**
Code files path error

Fixes #21019

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/AspNetCore.Docs?pullRequest=21000) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/AspNetCore.Docs?pullRequest=21000) before we can accept your contribution.<br/><br/>:x: wuyuansushen [sign now](https://cla.dotnetfoundation.org/dotnet/AspNetCore.Docs?pullRequest=21000)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/AspNetCore.Docs?pullRequest=21000) it.</sub>

File: aspnetcore/razor-pages/index/3.0sample/RazorPagesContacts/Pages/Customers/Create.cshtml.cs [hunk 1]
```diff
@@ -36,10 +36,10 @@ public async Task<IActionResult> OnPostAsync()
             _context.Customers.Add(Customer);
             await _context.SaveChangesAsync();
 
-            return RedirectToPage("./Index");
+            return RedirectToPage("/Index");
```
Comment (Rick-Anderson) [inline aspnetcore/razor-pages/index/3.0sample/RazorPagesContacts/Pages/Customers/Create.cshtml.cs:39]: I don't think this is right. This redirects to the root index, you want to redirect to the Customers Index.

**Review by Rick-Anderson — CHANGES_REQUESTED** [review]

File: aspnetcore/razor-pages/index/3.0sample/RazorPagesContacts/Pages/Customers/Create.cshtml.cs (hunk 1 above)
Comment (wuyuansushen) [reply aspnetcore/razor-pages/index/3.0sample/RazorPagesContacts/Pages/Customers/Create.cshtml.cs:39]: @Rick-Anderson I have known.

Comment (wuyuansushen) [reply aspnetcore/razor-pages/index/3.0sample/RazorPagesContacts/Pages/Customers/Create.cshtml.cs]: @Rick-Anderson I have corrected the error commit, other commits i think is no problem. Please review other commits and help to merge it.

**Pull request closed without merging**